axltempl/files/drupal/load.environment.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+// @codingStandardsIgnoreFile
 /**
  * This file is included very early. See autoload.files in composer.json and
```

Comment:
```suggestion
/**
 * @file
 * This file is included very early. See autoload.files in composer.json and
```

How about this instead?